components/Tabs/docs/tabbarview.md

```diff
@@ -0,0 +1,46 @@
+> **_NOTE:_** This is currently in Beta. Features may change without warning and without a change in the Material 
+> Components for iOS version number.
+
+### Importing MDCTabBarView
+
+`MDCTabBarView` is currently part of the MaterialComponentsBeta podspec. You may need to follow the [Material Components for iOS
+Beta integration guide](https://github.com/material-components/material-components-ios/blob/develop/contributing/beta_components.md) to configure
+your project to use `MDCTabBarView`.
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+import MaterialComponentsBeta.MaterialTabs_TabBarView
+```
+
+#### Objective-C
+
+```objc
+#import "MaterialTabs+TabBarView.h"
+```
+<!--</div>-->
+
+### Typical Use of MDCTabBarView
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+let tabBarView = MDCTabBarView()
+addSubview(tabBarView)
+
+configureTabBarViewConstraints()
```

Comment:
As a reader this method looks like something that might be part of the public API, but it looks like it's a placeholder for constraints logic. Can this be a comment instead?

Reply:
Done.